CMakeLists.txt

```diff
@@ -109,7 +105,11 @@ add_subdirectory(cmsis/device/rtos EXCLUDE_FROM_ALL)
 # Configures the application
 #
 function(mbed_configure_app_target target)
-    mbed_set_language_standard(${target})
+    target_compile_features(${target}
```

Comment:
shouldn't this be in our toolchain file (related to toolchain set up, although for an app)

Reply:
I reverted the changes and keep the `mbed_set_language_standard` funcion and moved `target_compile_feature` inside of it, please re-review